ee/clickhouse/queries/test/test_paths.py

```diff
@@ -1866,7 +1867,7 @@ def test_path_removes_duplicates(self):
             ],
         )
 
-    @test_with_materialized_columns(["$current_url"])
+    @test_with_materialized_columns(["$current_url", "$screen_name"])
```

Comment:
Somehow this just started to fail when it should've been failing all along

Reply:
Might have been a test ordering thing? Mat columns were not cleaned up between tests, fixed in a in-progress PR